iml-agent/src/action_plugins/lamigo.rs

```diff
@@ -168,31 +149,13 @@ mod lamigo_tests {
             cold_pool: "SLOW_POOL".into(),
             min_age: 35353,
             mountpoint: "/mnt/spfs".into(),
-            mailbox: "mailbox".into(),
+            mailbox_extend: "mailbox-extend".into(),
+            mailbox_resync: "mailbox-resync".into(),
         };
 
         assert_eq!(
             format_lamigo_conf_file(&config, "/etc/lamigo/{fs}-{mdt}.conf").unwrap(),
             PathBuf::from("/etc/lamigo/LU_TEST1-MDT0010.conf")
         )
     }
-
-    #[tokio::test]
-    async fn test_works() {
```

Comment:
Why remove the test?

Reply:
because we no longer generate lamigo service files.